testutil/network/network.go

```diff
@@ -557,7 +557,8 @@ func New(l Logger, baseDir string, cfg Config) (NetworkI, error) {
 			WithAccountRetriever(cfg.AccountRetriever).
 			WithAddressCodec(cfg.AddressCodec).
 			WithValidatorAddressCodec(cfg.ValidatorAddressCodec).
-			WithConsensusAddressCodec(cfg.ValidatorAddressCodec)
+			WithConsensusAddressCodec(cfg.ValidatorAddressCodec).
```

Comment:
The looks is right about this

Reply:
my bad, but is it ok to port back to 0.47.x as well since we have some [test](https://github.com/crypto-org-chain/chain-main/blob/b1f8504b05a6ec980b4ca76681cbbc40a6487587/x/nft/client/cli/cli_test.go#L63) need it

Reply:
You haven't introduced this, and it is only was an issue on main (address codecs aren't in the context in earlier versions).
Looks fine to backport this PR however.